spyder/config/main.py

```diff
@@ -307,6 +307,9 @@
               'console/use_project_or_home_directory': False,
               'console/use_cwd': True,
               'console/use_fixed_directory': False,
+              'console/fixed_directory': '',
+              'startup/fixed_directory': '',

```

Comment:
@steff456 since these values are empty we probably do not need to add them here
```
              'console/fixed_directory': '',
              'startup/fixed_directory': '',

```